### DISCUSSION TITLE
Getting the latest version using Conda

### DISCUSSION THREAD

**Discussion by danielzaidman** [rdkit/rdkit#4453] [Q&A]
Dear RDKit community,

I'm trying to update my version to the last version (which I believe is 2021.03.01).
I've tried a simple "conda install -c rdkit rdkit", and it said that everything is already installed.
But when I try to check the version in a python shell, I get the following output:

>>> from rdkit import rdBase
>>> rdBase.rdkitVersion
'2020.03.3'

What should I do?
Thanks, Daniel.

**Comment by greglandrum** [comment]
Hi Daniel,
The RDKit channel builds are no longer kept up-to-date.
We are now recommending that people install the RDKit from the conda-forge channel:
`conda install -c conda-forge rdkit`

**Comment by danielzaidman** [comment]
Thanks for you quick answer Greg.
However, I've tried it, and I get the same result...
Any other ideas?

Thanks, Daniel.

**Reply by greglandrum** [reply]
The first thing to try is to provide a specific version:
`conda install -c conda-forge rdkit='2021.03.5'`
If that doesn't work it's probably because of something else in your environment. The easiest solution then is to just start with a new environment
